app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -174,6 +176,10 @@ class HomeFragment : Fragment() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         postponeEnterTransition()
+        context?.runOnlyInMainProcess {
```

Comment:
More of a question than a request for change, why the `runOnlyInMainProcess` call here, is there a specific reason?

Reply:
Originally, the FenixApplication class was enabling the strict mode inside the `setupInMainProcessOnly` function. since we are also calling the `enableStrictMode` from HomeFragment, I figured they should be called under same condition?

Reply:
So I don't think this is to be worried since `HomeFragment.onCreate` should always be on the MT 😄  I'd go with removing `runOnlyInMainProces` in this case. iirc the reason why we called `setupInMainProcessOnly` is that there's option to offload to background threads in our application start and we can't do that with some of our stuff.